services/rabbit/enabled_plugins

```diff
@@ -0,0 +1 @@
+[rabbitmq_management, rabbitmq_management_agent, rabbitmq_web_dispatch, rabbitmq_prometheus].
```

Comment:
what about the versions of those plugins? Are they shipped with the version you froze or the service downloads them?

Reply:
yes, they are shipped with the version. that is why I had to update to 3.8.0 in order to get the prometheus endpoint.